vita3k/modules/SceProcessmgr/SceProcessmgr.cpp

```diff
@@ -190,26 +204,35 @@ EXPORT(int, sceKernelLibcGettimeofday, VitaTimeval *timeAddr, VitaTimezone *tzAd
     return 0;
 }
 
-EXPORT(Ptr<struct tm>, sceKernelLibcGmtime_r, const VitaTime *time, Ptr<struct tm> date) {
+EXPORT(Ptr<VitaTM>, sceKernelLibcGmtime_r, const VitaTime *time, Ptr<VitaTM> date) {
     TRACY_FUNC(sceKernelLibcGmtime_r, time, date);
     const time_t plat_time = *time;
 
-    SAFE_GMTIME(&plat_time, date.get(emuenv.mem));
+    auto dateIn = date.get(emuenv.mem);
+
+    tm host_tm;
+    memcpy(&host_tm, &dateIn, sizeof(VitaTM));
+    SAFE_GMTIME(&plat_time, &host_tm);
+    memcpy(dateIn, &host_tm, sizeof(VitaTM));
```

Comment:
I'm not sure if we need to clear host_tm before use in case of sizeof(struct tm)>sizeof(VitaTM) Something like
`if constexpr sizeof(struct tm)>sizeof(VitaTM) memset(&host_tm+sizeof(VitaTM),0,sizeof(struct tm)-sizeof(VitaTM));`

Reply:
Yes, I agree that this is needed, no need for the if tho, a memset of size 0 shouldn't cause any issue.

Reply:
Actually, doing 
``` 
tm host_tm = {};
```
should be fine and look better

Reply:
should i init with `{ 0 }` or `{}`?

Reply:
{0} is the same as {}